src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,3 +1,30 @@
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+import { Comment } from '../../types/Comment';
+
+type Props = {
+  comment: Comment,
+};
+
+export const CommentInfo: React.FC<Props> = (props) => {
+  const { comment: { name, email, body } } = props;
```

Comment:
```suggestion
export const CommentInfo: React.FC<Props> = ({ comment }) => {
  const { name, email, body } = comment;
```

I think this way would be more simple 🤔